tests/test_asset.py

```diff
@@ -30,6 +34,45 @@ def test_init(asset_mock):
     assert asset_mock.asset_id == ASSET_ID
 
 
+def test_repr(auth_mock):
+    asset_info = {"id": ASSET_ID}
+    asset = Asset(auth_mock, asset_info=asset_info)
```

Comment:
from unittest.mock import MagicMock
unreachable=MagicMock()

Reply:
I replaced the auth_mock with the MagicMock, I guess to isolate further the test given that auth_mock is not required to check if asset_info was correctly assigned. Please let's check if that was the idea for this suggestion.